frb/surveys/des.py

```diff
@@ -13,8 +16,7 @@
     print("Warning:  You need to install pyvo to retrieve DES images")
     _svc = None
 else:
-    _DEF_ACCESS_URL = "https://datalab.noao.edu/sia/des_dr2"
-    _svc = sia.SIAService(_DEF_ACCESS_URL)
+    _svc = sia.SIAService(defs.NOIR_DEF_ACCESS_URL)
```

Comment:
DES should not have the same access URL as DECaLs which is what `NOIR_DEF_ACCESS_URL` is currently defined as. Every survey has a different URL.

Reply:
Ah, got it.  My bad.  Fixing..